### PR TITLE
Review

### PR DESCRIPTION
**Pull request by cuongle1025**
Update:

- Added `addReview` in dbHandler and app.
- Added Frontend.js to deal with frontend api fetching.
- Added `getPokemonReviews, addReview` in Backend.js
- Modify dbhandler to fit with Pokemon.js function.
- Some minor changing.
- Implement Pokemon.js to have a review function.

User now can leave a review on a Pokemon:

- Name and picture of Pokemon are fetched from frontend api.
- Total reviews number by using backend api fetching `getPokemonReviews`.
- Display list of total reviews on Pokemon.
- If user has not had a review on Pokemon, leave a review button would show up to open review box.
- Show user review on Pokemon on page after submitting.

### CODE REVIEW

File: dbhandler.py [hunk 1]
```diff
@@ -109,10 +158,13 @@ def populate():
                 pokemon_list.append(models.Pokemon(pokedex_id=pokedex_id))
 
         for id in pokemon_list_ids:
-            rand_rating = randint(0, 10)
-            rand_user = randint(0, user_count-1)
-            review = models.Review(rating=rand_rating, title="Review of pokemon #" +
-                                   str(id), body="I may or may not like it :)")
+            rand_rating = randint(0, 5)
```
Comment (msantiago9) [inline dbhandler.py:161]: Interesting, so ratings are gonna be between 0-5 with mui. I will note that

File: package.json [hunk 2]
```diff
@@ -1,9 +1,12 @@
 {
   "dependencies": {
+    "@emotion/styled": "^11.3.0",
+    "@mui/material": "^5.1.0",
```
Comment (msantiago9) [inline package.json:4]: Do we need to do npm install <package> for this, or will "npm install" already install all of the dependencies?

File: src/Backend.js [hunk 3]
```diff
@@ -16,7 +17,25 @@ export const updateProfile = (username, img, bio) => {
...
+
+export const getUserReview = (username, id) =>
+  fetch('/getUserReview', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ username: username, pokemonid: id }),
+  })
+    .then((response) => response.json())
```
Comment (msantiago9) [inline src/Backend.js:38]: Nice, instead of returning the fetch you call fetch. I will modify Profile.js later with this.

File: src/Pokemon.js [hunk 4]
```diff
@@ -1,30 +1,112 @@
...
+      if (data === null) {
+        setUserReview(data);
+      } else {
+        setUserReview(data["reviews"][0]);
+      }
+    });
+  }, [setUserReview]
+  );
+  console.log(TotalReview);
+  console.log(UserReview);
```
Comment (msantiago9) [inline src/Pokemon.js:47]: Before sunday midnight we should hunt down all of our console.log statements and print statements and comment them out

File: package.json (hunk 2 above)
Comment (cuongle1025) [reply package.json:4]: Oh I forgot to update Readme. You need to npm install @mui/material and @emotion/styled for it to work.

File: src/Pokemon.js (hunk 4 above)
Comment (cuongle1025) [reply src/Pokemon.js:47]: Sure!

**Pull request merged**